packages/measurement-value/float-cdf.js

```diff
@@ -0,0 +1,42 @@
+const _ = require('lodash');
+
+const { MeasurementValue } = require('./measurement-value');
+
+class FloatCdf extends MeasurementValue {
+  xs(v) {
+    return v === undefined
+      ? _.get(this.input, 'xs')
+      : _.set(this.input, 'xs', v);
+  }
+
+  ys(v) {
+    return v === undefined
+      ? _.get(this.input, 'ys')
+      : _.set(this.input, 'ys', v);
+  }
+
+  sizeXs() {
+    return _.size(this.xs());
+  }
+
+  sizeYs() {
+    return _.size(this.ys());
+  }
+
+  validate() {
+    if (this.sizeXs() !== this.sizeYs()) {
+      throw new Error('Xs and Ys should be the same size.');
+    }
+    if (this.sizeXs() > FloatCdf.MAX_XS) {
```

Comment:
This should be a server thing, maybe configurable here, but not forced in this lib.

Reply:
I propose to forget about this for now, later we can fix it. Here is no much sense to write configuration providers and so on.

Reply:
Ok, sure